File: maddie-carlson/day-04/no-spaces.js
function noSpace(x) {
  // parameter is x, which is a string
  // should return another string with any and all spaces removed
  // this can be accomplished by replacing all spaces with an empty string with replaceAll()
  return x.replaceAll(" ", "");
}

/*
  Ember's Feedback:
  Great job!
  Good use of replaceAll
  it will iterate over your string and every time it finds the first parameter,
  it will replace it with the second parameter. It builds up a new string and returns it to you.
*/

File: cody-miller/day-31/no-spaces.py
def no_space(x):
    y = x.replace(" ", "")
    return y

# Replace the spaces with no spaces.

"""
    Ember's Feedback:
    - Good job!
    - make this file snake_case
"""


# Alternative Solution
def no_space(x):
    return "".join(x.split())


def no_space(x):
    import re

    return re.sub("\s*", "", x)
